src/models/post.js

```diff
@@ -50,3 +50,7 @@ export const denormalizedPost = (post, state) => ({
 })
 
 export const isMessageThread = post => get('type', post) === 'thread'
+
+export const isChildPost = post => get('parent_post_id', post)
```

Comment:
use !! here and below so that the values returned are true and false, not int and null